client/components/web-preview/toolbar.jsx

```diff
@@ -175,7 +175,7 @@ class PreviewToolbar extends Component {
 							href={ editUrl }
 							onClick={ this.handleEditorWebPreviewEdit }
 						>
-							{ translate( 'Edit' ) }
+							{ translate( 'Edit Homepage' ) }
```

Comment:
For button labels, sentence case is preferred. Mind updating `Edit Homepage` to use a lowercase `h`?